plugin-server/functional_tests/analytics-ingestion/error-handling.test.ts

```diff
@@ -33,19 +34,19 @@ beforeAll(async () => {
 })
 
 afterAll(async () => {
-    await dlqConsumer.disconnect()
+    await warningConsumer.disconnect()
 })
 
-test.concurrent('consumer handles messages just over 1MB gracefully', async () => {
```

Comment:
should we add another test for DLQ - unknown error of sorts?

Reply:
Yeah, I think it's a good idea to keep the DLQ path covered. I'll look and see if there's a relatively straightforward way of exercising that path.

Reply:
After poking around a bit more today, I'm not sure how much sense it makes to cover this as a functional test moving forward. We would need to be able to create a knowingly problematic event to send into the pipeline that causes an unhandled error -- but with changes like this one, we're intentionally reducing the number such events so that the DLQ will only contain events that triggered previously unanticipated errors.

The only way I can think of to test _that_ behavior would be to mock out parts of the pipeline so that they throw arbitrary unhandled exceptions on otherwise valid inputs, and ensure those events get produced to the DLQ -- and [that is already done here, just not as a functional test](https://github.com/PostHog/posthog/blob/286b6899984d4d481fd8a67755f655f7939c773c/plugin-server/tests/worker/dead-letter-queue.test.ts) due to needing to be able to inject the error.

Is there an alternative test method that you think would make sense here that I'm overlooking?

Reply:
Nope, just ship it